example/zping-server/zping-server.go

```diff
@@ -79,12 +79,16 @@ func main() {
 		identity, _ := context.GetCurrentIdentity()
 		fmt.Printf("\n%+v now serving\n\n", identity.Name)
 		listener, err = context.ListenWithOptions(service, &options)
```

Comment:
Probably either want to do logrus.FatalF, or panic, as panic will also emit the error message. Depends if you want the stack trace or not